Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/impl/ConceptManager.java

```diff
@@ -381,9 +381,9 @@ public String addConceptListEntry(ConceptEntry entry)
 			throw new DictionaryModifyException();
 		}
 
-		indexService.insertConcept(entry);
 		String id = generateId(CONCEPT_PREFIX);
```

Comment:
Also, unrelated to this story, but let's do it real quick anyways. Let's generate the ids a little differently, so that they are more readable. Let's generate short ids along the lines we do it in Quadriga now:
https://github.com/diging/quadriga/blob/develop/Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/BaseDAO.java